src/events/music/trackStart.ts

```
@@ -87,132 +91,144 @@
+                        ],
+                    }),
                 );
                 break;
-            case 'detailed':
+            case 'detailed': {
+                let thumbnailURL = `https://i.ytimg.com/vi/${track.info.identifier}/hqdefault.jpg`;
+                if (track.info.sourceName === 'deezer') {
+                    const { body } = await request(`https://api.deezer.com/track/${track.info.identifier}`);
+                    let dzTrackInfo;
+                    try {
+                        dzTrackInfo = await body.json() as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)